site/specs/messaging.yml

```diff
@@ -1849,7 +1849,179 @@ components:
+        application/pdf:
+          schema:
+            type: string
+            format: binary
+        application/rtf:
+          schema:
+            type: string
+            format: binary
+        application/zip:
+          schema:
+            type: string
+            format: binary
+        application/x-tar:
+          schema:
+            type: string
+            format: binary
+        application/xml:
+          schema:
+            type: string
+            format: binary
+        application/gzip:
+          schema:
+            type: string
+            format: binary
+        application/x-bzip2:
+          schema:
+            type: string
+            format: binary
+        application/x-gzip:
+          schema:
+            type: string
+            format: binary
+        application/smil:
+          schema:
+            type: string
+            format: binary
+        application/javascript:
+          schema:
+            type: string
+            format: binary
+        audio/mp4:
+          schema:
+            type: string
+            format: binary
+        audio/mpeg:
+          schema:
+            type: string
+            format: binary
+        audio/ogg:
+          schema:
+            type: string
+            format: binary
+        audio/flac:
+          schema:
+            type: string
+            format: binary
+        audio/webm:
+          schema:
+            type: string
+            format: binary
+        audio/wav:
+          schema:
+            type: string
+            format: binary
+        audio/amr:
+          schema:
+            type: string
+            format: binary
+        audio/3gpp:
+          schema:
+            type: string
+            format: binary
+        image/bmp:
+          schema:
+            type: string
+            format: binary
+        image/gif:
+          schema:
+            type: string
+            format: binary
+        image/jpeg:
+          schema:
+            type: string
+            format: binary
+        image/pjpeg:
+          schema:
+            type: string
+            format: binary
+        image/png:
+          schema:
+            type: string
+            format: binary
+        image/svg+xml:
+          schema:
+            type: string
+            format: binary
+        image/tiff:
+          schema:
+            type: string
+            format: binary
+        image/webp:
+          schema:
+            type: string
+            format: binary
+        image/x-icon:
+          schema:
+            type: string
+            format: binary
+        text/css:
+          schema:
+            type: string
+            format: binary
+        text/csv:
+          schema:
+            type: string
+            format: binary
+        text/calendar:
+          schema:
+            type: string
+            format: binary
+        text/plain:
+          schema:
+            type: string
+            format: binary
```

Comment:
Is `format: binary` okay for text types, or should it be something different?  The idea with this content-type is that user would upload a [.txt] file as with all types. But that's not a really a binary format

Reply:
It depends on what the user is uploading honestly. If users are uploading files (which is generally the case), then `format: binary` is the correct way to define it in OpenAPI 

Otherwise the other content types should reflect the correct request body format 

For text/plain you could drop the binary - but what would the content type be for someone uploading a .txt file?

Reply:
I was confused on that, whether uploading a .txt file produces a different HTTP request than just sending raw text.